### ISSUE TITLE
When are the changes uploaded to the repo?

### ISSUE THREAD

**Issue by jbagot** [bitnami/charts#3263]
Hi, I just created a PR and it has been merged. When will be uploaded in the repo?
I don't know if this is posted somewhere but I didn't find it.
If this information is not published maybe it's interesting to add it in the README.

**Comment by miguelaeh** [comment]
Hi @jbagot ,
Once the PR is merged they take some hours to be published to the helm registry. After merging the PR our internal pipeline takes the new version and performs some tests, like application tests and upgrade tests.
It should be published in a few hours if it is not please provide us with the PR number.

**Comment by jbagot** [comment]
Hi, it's already published. My comment was more about share the information adding it in the README or somewhere because yesterday I was updating the repo all the time until it appeared, would be nice to have this information IMHO.
Thank you for your reply :+1:

**Comment by miguelaeh** [comment]
Hi @jbagot ,
Thanks for you suggestion we will take it into account.

**Comment by miguelaeh** [comment]
Hi @jbagot ,
I was looking where to add that description but I found it already in the following file: https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md

**Comment by jbagot** [comment]
Hi @miguelaeh ,
Ok, thank you I didn't see it. Thank you to mention it. I close the issue.
